sdk/aqueduct/utils.py

```diff
@@ -110,9 +110,11 @@ def retention_policy_from_latest_runs(k_latest_runs: int) -> RetentionPolicy:
 
 MODEL_FILE_NAME = "model.py"
 MODEL_PICKLE_FILE_NAME = "model.pkl"
+FUNCTION_SOURCE_FILE_NAME = "source.py"
```

Comment:
@vsreekanti Is it too restrictive to make `source.py` a reserved file?

Reply:
I would prefer not to. Would it be possible to make the filename in the zipfile the name of the function?

Reply:
Sure, we'll just have to prevent users from uploading file dependencies with the same name as the function.

Reply:
Yep, I think that's fine.

Reply:
Fixed, the files now look like:
<img width="839" alt="Screen Shot 2022-08-29 at 4 48 40 PM" src="https://user-images.githubusercontent.com/10413474/187318183-e461b4e1-9950-43e6-95b8-9301c3931a55.png">

Reply:
Yep, that's perfect.